src/mpi/romio/adio/common/lock.c

```diff
@@ -5,7 +5,7 @@
 
 #include "adio.h"
 
-static const char *flock_cmd_to_string(int cmd)
+const char *flock_cmd_to_string(int cmd)
```

Comment:
Is there a reason to remove the static label for these?

Reply:
I want to call the same function in Quobyte lock implementation:
https://github.com/pmodels/mpich/blob/51a9278121738dbe695dde148486440ec25c55de/src/mpi/romio/adio/ad_quobytefs/ad_quobytefs_setlock.c#L36

Would you want me to implement it as static in a header file?

Reply:
The concern is the namespace.  If those symbols are exposed, the namespace would need to be cleaner.  Can you also separate that out into a separate patch, because that's existing code that you are modifying?

Reply:
Done please review  #4551

Reply:
Thanks.  #4451 does match the spirit of what I was suggesting, except for the namespace.  All symbols that are not static need to be within our reserved namespace.